Error when trying to include service

I'm trying to call the TriggerSnapshot service from within my ros node, but when I include the service in my node like this:

`#include <rosbag_snapshot_msgs/TriggerSnapshot.h>`

without adding any further code, I get the following error. I have added rosbag_snapshot_msgs to my CMakeLists and package.xml. Any idea what could be causing this?


`/opt/ros/noetic/include/rosbag_snapshot_msgs/TriggerSnapshotResponse.h:52:125: error: expected unqualified-id before numeric constant
   52 |   static const std::basic_string<char, std::char_traits<char>, typename ContainerAllocator::template rebind<char>::other >  NO_DATA;
      |                                                                                                                             ^~~~~~~
/opt/ros/noetic/include/rosbag_snapshot_msgs/TriggerSnapshotResponse.h:68:53: error: expected unqualified-id before numeric constant
   68 |       TriggerSnapshotResponse_<ContainerAllocator>::NO_DATA =
      |                                                     ^~~~~~~
`

--- a/rosbag_snapshot/src/snapshotter.cpp
+++ b/rosbag_snapshot/src/snapshotter.cpp
@@ -428,7 +428,7 @@ bool Snapshotter::triggerSnapshotCb(rosbag_snapshot_msgs::TriggerSnapshot::Reque
   if (!bag.isOpen())
   {
     res.success = false;
-    res.message = res.NO_DATA;
+    res.message = res.NO_DATA_MESSAGE;
     return true;
   }
 

--- a/rosbag_snapshot/test/test_snapshot.py
+++ b/rosbag_snapshot/test/test_snapshot.py
@@ -90,7 +90,7 @@ class TestRosbagSnapshot(unittest.TestCase):
         filename = tempfile.mktemp()
         res = self.trigger(filename=filename, topics=topics)
         self.assertFalse(res.success)
-        self.assertEqual(res.message, TriggerSnapshotResponse.NO_DATA)
+        self.assertEqual(res.message, TriggerSnapshotResponse.NO_DATA_MESSAGE)
         self.assertFalse(os.path.isfile(filename))
 
     def _assert_record_success(self, data):

--- a/rosbag_snapshot_msgs/srv/TriggerSnapshot.srv
+++ b/rosbag_snapshot_msgs/srv/TriggerSnapshot.srv
@@ -14,5 +14,5 @@ time stop_time    # Latest timestamp for a message to be included in written bag
 
 bool success    # True if snapshot successfully written to disk
 
-string NO_DATA=no messages buffered on selected topics
+string NO_DATA_MESSAGE=no messages buffered on selected topics
 string message  # Error description if failed

